examples/example_sdl_opengl3/Makefile

```diff
@@ -47,9 +61,16 @@ CXXFLAGS += -I../libs/gl3w
 
 ifeq ($(UNAME_S), Linux) #LINUX
 	ECHO_MESSAGE = "Linux"
-	LIBS += -lGL -ldl `sdl2-config --libs`
 
+ifndef RPI
```

Comment:
I am not aware of Makefile conventions, but indentation here does look confusing. Do people treat `ifdef`/`endif` in Makefiles as `#ifdef` or as `if` in c?

Reply:
As far as I know, Makefile syntax in this case does not impose the rules about indents (just like C does not). IMO, it is more a matter of convention and what original code uses.
I used indent as they were used before my change. I should have followed this approach for the `ifndef` on lines 32 and 40 as well to be consistent.